tests/models/test_identifier.py

```diff
@@ -358,28 +387,66 @@ def test_recursively_equivalent_identifier_ids(self):
         l2.equivalent_to(data_source, another_identifier, 1)
         l3.equivalent_to(data_source, l2, 1)
         l4.equivalent_to(data_source, l3, 0.9)
+        high_levels_fairly_high_threshold = PresentationCalculationPolicy(
+            equivalent_identifier_levels=5,
+            equivalent_identifier_threshold=0.89
+        )
         equivs = Identifier.recursively_equivalent_identifier_ids(
-            self._db, [another_identifier.id], levels=5, threshold=0.89)
+            self._db, [another_identifier.id],
+            high_levels_fairly_high_threshold
+        )
         eq_(set([another_identifier.id,
                  l2.id,
                  l3.id,
                  l4.id]),
             set(equivs[another_identifier.id]))
 
         # We can look for multiple identifiers at once.
+        two_levels_high_threshold = PresentationCalculationPolicy(
+            equivalent_identifier_levels=2,
+            equivalent_identifier_threshold=0.8
+        )
         equivs = Identifier.recursively_equivalent_identifier_ids(
-            self._db, [identifier.id, level_3_equivalent.id], levels=2, threshold=0.8)
+            self._db, [identifier.id, level_3_equivalent.id], 
+            policy=two_levels_high_threshold
+        )
         eq_(set([identifier.id,
                  strong_equivalent.id]),
             set(equivs[identifier.id]))
         eq_(set([level_2_equivalent.id,
                  level_3_equivalent.id]),
             set(equivs[level_3_equivalent.id]))
 
-        # The query uses the same db function, but returns equivalents
-        # for all identifiers together so it can be used as a subquery.
+        # By setting a cutoff, you can say to look deep in the tree,
+        # but stop looking as soon as you have a certain number of
+        # equivalents.
```

Comment:
Something weird is going on here. The `fn_recursive_equivalents` function, defined in model/files/recursive_equivalents.sql, takes `cutoff` as an argument but doesn't do anything with it. But these tests show that setting a higher `cutoff` does make a difference. It can't be some old version of  `fn_recursive_equivalents` embedded in my database because 1) I checked its definition and 2) the tests pass on Travis.

Reply:
The cutoff is confusingly referred to as `$4` in the function.

Reply:
Whew!